la-societe-francaise-detude-de-la-ceramique-antique-en-gaule.csl

```diff
@@ -0,0 +1,303 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="never" page-range-format="expanded" default-locale="fr-FR">
+  <info>
+    <title>La Société Française d’Étude de la Céramique Antique en Gaule (French)</title>
+    <title-short>SFECAG</title-short>
+    <id>http://www.zotero.org/styles/la-societe-francaise-detude-de-la-ceramique-antique-en-gaule</id>
+    <link href="http://www.zotero.org/styles/la-societe-francaise-detude-de-la-ceramique-antique-en-gaule" rel="self"/>
+    <link href="http://www.zotero.org/styles/bibliotheque-d-archeologie-mediterraneenne-et-africaine-biama" rel="template"/>
+    <link href="http://ccj.cnrs.fr/spip.php?rubrique89" rel="documentation"/>
```

Comment:
Should be http://sfecag.free.fr/recommandations.pdf. @POBrien333, can you change the setting on this pull request so I can edit it?